src/WebJobs.Extensions.DurableTask/ActorScheduler/ResponseMessage.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.ExceptionServices;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class ResponseMessage
+    {
+        public string Result { get; set; }
```

Comment:
Is this property intended to be serialized? If so, let's decorate all members, similar to how you are doing with the `OperationMessage` class.